scripts/primaza.sh

```diff
@@ -73,10 +70,7 @@ function deploy() {
       --set app.image=${PRIMAZA_IMAGE_NAME} \
       --set app.host=${INGRESS_HOST} \
       --set app.envs.git.sha.commit=${GIT_SHA_COMMIT} \
-      --set app.envs.github.repo=${PRIMAZA_GITHUB_REPO} \
-      --set app.envs.VAULT_URL=${VAULT_URL} \
-      --set app.envs.VAULT_USER=${VAULT_USER} \
-      --set app.envs.VAULT_PWD=${VAULT_PWD} \
```

Comment:
I prefer that we keep them part of the bash script which is taking care to install the `primaza-app` helm chart otherwise the helm chart will override the env var with empty values (see templates/deployment.yaml file):
```
    spec:
      containers:
        - env:
            ...
            - name: vault.password
              value: {{ .Values.app.envs.vault.password }}
            - name: vault.user
              value: {{ .Values.app.envs.vault.user }}
            - name: vault.url
              value: {{ .Values.app.envs.vault.url }}
```
Otherwise, it is needed that we have some `{{- if}}` statements part of the deployment.yaml to include within the pod container, the vault env. If no helm values `.Values.app.envs.vault.xx` are declared, then no block loke this will be included within the deployment.yaml
```
            - name: vault.xx
              value: {{ .Values.app.envs.vault.xx }}
```
Can we do that ? @Sgitario

Reply:
Adding if statements in the middle of a resource is not supported.
An alternative solution is to provide a profile that sets these values to empty or disabling vault by default (doing `${vault.url:}`, ...).

Reply:
No. The current configuration: 
```
# Vault configuration
quarkus.vault.url=${vault.url:http://vault.127.0.0.1.nip.io}
quarkus.vault.authentication.userpass.username=${vault.user:bob}
quarkus.vault.authentication.userpass.password=${vault.password:sinclair}
```
is generating correctly the appropriate values.yml:
```
app:
  envs:
    vault:
      password: sinclair
      user: bob
      url: http://vault.127.0.0.1.nip.io
```
And deployment.yml is using them: 
```
            - name: vault.url
              value: {{ .Values.app.envs.vault.url }}
```
For that : ` mvn clean install -DskipTests -Pkubernetes`